app/assets/stylesheets/components/_subheader.scss

```diff
@@ -3,8 +3,13 @@
 }
 
 .global-subheader {
-  padding-top: $container-v-padding;
+  padding-top: 12px;
+  padding-bottom: 12px;
```

Comment:
How come you're explicitly stating these? Wouldn't it be better to use a shorthand?